test/models/user_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+    @github_omniauth_hash = OmniAuth.config.mock_auth[:github]
+    @user                 = users(:jim)
+  end
+
+  context 'validations' do
```

Comment:
I'm torn about testing validations, we know ActiveRecord is well tested so these should work if they're in your `User` class. This is essentially testing that you called those methods as you expected.

I used to want to test everything like this but I don't usually do it anymore. Up to you if you want to keep these tests or not.